Taskfile.yml

```diff
@@ -63,9 +63,9 @@ tasks:
       - cmd: docker buildx prune --all
         ignore_error: true # build container may not exist
       - docker system prune --all --volumes
-      - bin/golangci-lint{{exeExt}} cache clean
+      - bin/golangci-lint cache clean
       - go clean -cache -testcache -modcache -fuzzcache
-      - rm -fr .cache .task tools/.task integration/.task tmp
```

Comment:
Removing `-f` will error with non-zero exist code on non-existing files and directories. So it won't always work. We wouldn't want to remove it.

Less important thing, it will prompt on write protected files, making it a bit more manual to delete.